docs/roles/project-coordinator.rst

```diff
@@ -8,32 +7,43 @@ project lead for a specific project. This page explains more about the
 responsibilities and role of the project coordinator.
 
 
-Description
+Description 
 -----------
 
-The project coordinator is responsible for ensuring that all club projects have the resources needed to continue developing. This may include monetary resources, connections to mentors or the like and infrastructure to host the projects. 
+The project coordinator is responsible for ensuring that all club projects have
+the resources needed to continue developing.  This may include monetary
+resources, connections to mentors or the like and infrastructure to host the
+projects. 
 
 Eligibility requirements
 ^^^^^^^^^^^^^^^^^^^^^^^^
 
-The project coordinator must maintian good academic standing as defined by RIT to remain a member of eboard.
-Due to the hands on nature of the role, the project coordinator must be available in person during club and eboard meetings. If needed, a temporary replacement may be appointed to fill the role while the project coordinator is out on coop. In that case there should be good communication between the the two people in order to facilitate a smooth transition.
+The project coordinator must maintain good academic standing as defined by RIT
+to remain a member of eboard.  Due to the hands on nature of the role, the
+project coordinator must be available in person during club and eboard
+meetings.  If needed, a temporary replacement may be appointed to fill the role
+while the project coordinator is out on coop. In that case there should be good
+communication between the the two people in order to facilitate a smooth 
+transition.
 
 
-Responsibilities
+Responsibilities 
 ----------------
 
-* Be physically present at all eboard meetings
-* Be present at all RITLUG meetings unless a conflict has come up and eboard has been notified
-* Be available to RITLUG members
-* Act as liason between project leads and the rest of eboard
-* Assist with general eboard tasks as needed
+- Act as liason between project leads and the rest of eboard 
+- Mediate disputes between project leads and team members
+- Encourage members to start and participate in club projects
+- Assist project leads in managing their projects
+- Assist project leads in establishing project specific meetings
+- Provide general support to project leads and participants
```

Comment:
Amazing. This is great and is super helpful, and it's much more specific to the project coordinator role. 😄